draft-ietf-httpbis-cache-latest.xml

```diff
@@ -1237,10 +1234,9 @@
 </t>
 <t>
    The presence of an Age header field implies that the response was not
-   generated or validated by the origin server for this request. However,
-   lack of an Age header field does not imply the origin was contacted, since
-   the response might have been received from an HTTP/1.0 cache that does not
-   implement Age.
+   generated or validated by the origin server for this request. Note that
+   in rare circumstances, the response might have been received from an HTTP/1.0
+   cache that does not implement Age.
```

Comment:
```suggestion
   generated or validated by the origin server for this request.
```

Reply:
We need to either keep the original or remove the sentence entirely -- the entire point of that sentence is to say that lack of Age doesn't imply the reverse (that the response was generated or validated by the origin server for this request). I'd be fine with deleting or keeping the original sentence, or just removing the explanatory suffix ", since the response might ..."

Reply:
I mean, the original sentence:

> However, lack of an Age header field does not imply the origin was contacted, since the response might have been received from an HTTP/1.0 cache that does not implement Age.

can be deleted or replaced with

> However, lack of an Age header field does not imply the origin was contacted, since the response might have been received from a non-conforming cache.

or simply

> However, lack of an Age header field does not imply the origin was contacted.

Reply:
See latest comment; I went with the last suggestion. Thanks.